templates/mobile/activation/activation.1.schema.json

```diff
@@ -0,0 +1,57 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "title": "activation",
+  "properties": {
+    "identifier": {
+      "type": "string",
+      "description": "hashed version of Google ID"
+    },
+    "client_id": {
+      "type": "string",
+      @COMMON_PATTERN_UUID_1_JSON@,
+      "description": "telemetry client_id, included only if identifer is empty"
+    },
+    "manufacturer": {
+      "type": "string",
+      "description": "Build.MANUFACTURER"
+    },
+    "model": {
+      "type": "string",
+      "description": "Build.MODEL"
+    },
+    "distribution_id": {
+      "type": "string"
+    },
+    "locale": {
+      "type": "string",
+      "description": "application locale, such as 'en-US'"
+    },
+    "os": {
+      "type": "string"
+    },
+    "osversion": {
+      "type": "string"
+    },
+    "created": {
+      "type": "string"
```

Comment:
Does our schema matching implementation allow the use of `format`? If so, seems like this could use `date-time`

Reply:
json schema v4 does allow specifying "format": "date-time". The validator that hindsight uses does not support checking formats, but the one we're using in the GCP pipeline does. I think there's a good possibility this ping will be able to skip the old pipeline, so perhaps this is indeed worth specifying here.

Reply:
Per the client-side implementation (https://phabricator.services.mozilla.com/D29668), this string is going to be in the format "YYYY-MM-DD" without a time within the day. JSON schema v7 includes a "data" format that corresponds to this, but the "date-time" of v4 is limited to strings that include both date and time. So going to leave this as a simple string.